src/components/AppForm.js

```diff
@@ -0,0 +1,92 @@
+import { LitElement, html, css } from 'lit-element';
+
+import './AppButton';
+
+class AppForm extends LitElement {
+
+  static get styles() {
+    return css`
+      .wrapper {
+        padding: 20px;
+        border: 1px solid #eee;
+        background-color: #fff;
+        border-radius: 6px;
+      }
+      div {
+        padding: 6px 0;
+      }
+      input[type="text"], textarea {
+        width: 100%;
+        padding: 6px 8px;
+        box-sizing: border-box;
+      }
+      textarea {
+        resize:vertical;
+      }
+    `;
+  }
+
+  constructor() {
+    super();
+
+    this.title = '';
+    this.description = '';
+  }
+
+  /**
+   * Handle change in input value.
+   * @param {Object} e Input event object. 
+   */
+  handleInputChange = (e) => {
+    this[e.target.name] = e.target.value;
+  }
+
+  handleAddTodo = () => {
+
+    if (this.title) {
+      const todo = {
+        id: Date.now(),
+        title: this.title,
+        description: this.description,
+        active: true
+      }
+
+      this.title = '';
+      this.description = '';
+
+      this.addHandler(todo);
+    }
+  }
+
+  render() {
+    return html`
+      <div class="wrapper">
+        <form>
+          <div>
+            <label>Title</label>
+            <div>
+              <input type="text" name="title" @input=${this.handleInputChange}>
```

Comment:
I was expecting you to use either `paper` or `mwc` web components instead of native HTML tag here. :smile:

Reply:
Also, make a habit of using: 

```
@input="${this.handleInputChange}">

```